pages/en/network/developing.mdx

```diff
@@ -0,0 +1,55 @@
+
+### Curation
+
+Curators use their knowledge of web3 to assess and signal new subgraphs. They look for subgraphs with reliable and relevant data.
+
+Curators stake GRT with the subgraph and receive curation shares, which guarantee a percentage of future query fees. The curation signal increases as more Curators signal on the subgraph.
+
+A strong curation signal is crucial to a subgraph. Indexers use the curation signal to determine which subgraphs to index to the network, making it available for querying.
+
+### Deploying on the network
+
+By deploying your subgraph to The Graph network, you're making that data available to developers and users across the web3 ecosystem. Anyone will be able to query it.
+
+Data from your subgraph can be used to improve user experience, build new features, or even create entirely new applications.
+
+## Querying Subgraphs
+
+Developers that query subgraphs serve it to users through a simple interface. They can use the GraphQL API to query data directly from the blockchain. Developers can manipulate, sort, or aggregate the data any way they want.
+
+Querying a subgraph with your API key will generate query fees that can be paid in GRT.
+
+### How Billing Works
+
+Developers that query a subgraph can use GRT to pay the query fees. This fee is split between different network participants, including the Indexers, Curators, and Delegators.
+
+Your account on [Subgraph Studio](https://thegraph.com/studio/) must have a positive balance of GRT, which will be used to pay for query fees to the network. Maintaining a positive balance for an account is the responsibility of the user or dapp.
+
+There are two ways to be notified of your query fee balance. The first is to set up a billing alert that will be sent to an [email address](https://thegraph.com/studio/settings/). Another way is by querying the [billing subgraph](https://thegraph.com/hosted-service/subgraph/graphprotocol/billing?query=Nena%20account) API directly.
+
+Learn how to prioritize price, speed, data freshness (among other factors) for your queries by watching this video.
+
+<VideoEmbed src="https://www.loom.com/embed/b5fc533e48584cb694017392c80c75e0" />
+
+> IMPORTANT: When an account reaches a 0 GRT balance and the billing invoice is not paid in a timely manner, indexers will stop serving queries for all API keys associated with the account. Indexers will automatically start serving queries again once the billing balance is paid in full and GRT is added to the account for future query fees.
+
+### Paying Query Fees with GRT
```

Comment:
This section seems more fitting for a Managing subgraphs on the network section, vs. a description of the role developers play in the network?

Reply:
I totally agree - removing this section.